src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,392 @@
+
+**Videos:**
+
+- [egghead.io: MobX](https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx)
+
+**Tutorials:**
+
+- [The Modern JavaScript Tutorial](https://javascript.info/)
+- [Official Node.js Tutorial](https://nodejs.dev/learn)
+- [egghead.io: A Beginner's Guide to React](https://egghead.io/courses/the-beginner-s-guide-to-react)
+- [Official Webpack Tutorial](https://webpack.js.org/guides/getting-started/)
+
+**Resources:**
+
+- [MDN Web Docs: JavaScript Resources](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+- [Official TypeScript Documentation](https://www.typescriptlang.org/docs/handbook/)
+- [Official React Documentation](https://reactjs.org/)
+- [Official MobX Documentation](https://mobx.js.org/README.html)
+- [Official Gatsby Documentation](https://www.gatsbyjs.com/docs/)
+- [Official Tailwind Documentation](https://tailwindcss.com/docs)
+
+</details>
+
+<details>
+<summary> Web Development (Internal) </summary>
+
+**Reading:**
+
+- [NUbook: NUsight](/system/tools/nusight)
+
+**Resources:**
+
+- [GitHub: NUsight](https://github.com/NUbots/NUbots/tree/master/nusight2)
+- [GitHub: NUbook](https://github.com/NUbots/NUbook)
+
+</details>
+
+<details>
+<summary> Documentation </summary>
+
+Since NUbots is a student organisation, high turnover is unavoidable. To help combat the issues this creates, good documentation is key to the success of the team. This includes low level documentation in the form of code comments and high level documentation in the form of NUbook pages.
+
+**Reading:**
+
+- [NUbook: NUbook](/system/tools/nubook)
+- [Australian Network on Disability: A Beginner's Guide to Accessible Content](https://www.and.org.au/pages/a-beginners-guide-to-accessible-content.html)
+
+**Tutorials:**
+
+- [Google Technical Writing Course](https://developers.google.com/tech-writing) - free, in-class sessions are open to the public!
+
+</details>
+
+## Hardware
+
+<details>
+<summary>Soccer Robot</summary>
+
+**Reading:**
+
+- [NUbook: Hardware Overview and Specifications](/system/hardware/overview)
+- [NUbook: Input](/system/subsystems/input)
+- [NUbook: Flashing a Robot](/guides/hardware/flashing)
+
+**Resources:**
+
+- [GitHub: NUware](https://github.com/NUbots/NUware)
+
+</details>
+
+## Localisation and Odometry
+
+<details>
+<summary> Particle Filters </summary>
+
+**Videos:**
+
+- [YouTube: Introduction to Autonomous Navigation](https://www.youtube.com/watch?v=Fw8JQ5Q-ZwU)
+- [YouTube: Understanding the Particle Filter](https://www.youtube.com/watch?v=NrzmH_yerBU)
+- [YouTube: Monte Carlo Localisation and Particle Filters](https://www.youtube.com/watch?v=MsYlueVDLI0&t=1608s)
```

Comment:
is this link meant to be timestamped?

Reply:
I believe so. @abihall ?

Reply:
They aren't meant to be timestamped, the whole video is good